mix.exs

```diff
@@ -257,10 +257,11 @@ defmodule Livebook.MixProject do
       "docs/deployment/fips.md",
       "docs/deployment/nginx_https.md",
       "docs/teams/intro_to_teams.md",
-      "docs/teams/shared_secrets.md",
-      "docs/teams/shared_file_storages.md",
       {"docs/teams/email_domain.md", title: "Email domain auth"},
       {"docs/teams/oidc_sso.md", title: "OIDC SSO"},
+      {"docs/teams/oidc_groups.md", title: "OIDC groups authorization"},
```

Comment:
Don't we read the title automatically from the actual file? Why are we passing it here?

Reply:
Good catch. Fixed in https://github.com/livebook-dev/livebook/pull/2996/commits/ba3d5c47e0dffecbac9b528fee0ba674d921040f